index.bs

```diff
@@ -2916,6 +2916,246 @@ value and terminate the operation.
+       1. If the result of [=base64url encoding | base64url decoding=]
+           |credentialId| is an error, then throw a {{TypeError}}.
+
+The [=currentCredentials=] [=credential report/authenticator actions=] take a
+{{CurrentCredentialsReport}} |options| and are as follows:
+1. Let |userId| be result of [=base64url encoding | base64url decoding=]
+    <code>|options|.{{CurrentCredentialsReport/userId}}</code>.
+1. Assertion: |userId| is not an error.
+1. Let |credential| be
+    <code>[=credentials map=][|options|.{{CurrentCredentialsReport/rpId}}, |userId|]</code>.
+1. If |credential| does not exist, abort these steps.
+1. If <code>|options|.{{CurrentCredentialsReport/user}}</code> is present,
+    update the |credential|'s [=public key credential source/otherUI=] to match
+    <code>|options|.{{CurrentCredentialsReport/user}}.{{CurrentCredentialUserDetails/name}}</code>
+    and
+    <code>|options|.{{CurrentCredentialsReport/user}}.{{CurrentCredentialUserDetails/displayName}}</code>
+1. If
+    <code>|options|.{{CurrentCredentialsReport/allAcceptedCredentialIds}}</code>
+    is present and
+    <code>|options|.{{CurrentCredentialsReport/allAcceptedCredentialIds}}</code>
+    does NOT [=list/contain=] the result of [=base64url encoding=] the
+    |credential|'s [=public key credential source/id=], then [=map/remove=]
+    |credential| from the [=credentials map=] or employ an
+    [=authenticator=]-specific procedure to hide it from future [=authentication
+    ceremonies=].
+
+<div class="example">
+
+A user updates their name on a [=[WRP]=] provided UI. The [=[WRP]=] runs:
+
+```javascript
+PublicKeyCredential.signal({
+    currentCredentials: {
+        rpId: "example.com",
+        userId: "aabbcc",  // user handle, base64url.
+        user: {
+            name: "New user name",
+            displayName: "New display name"
+        }
+    }
+});
+```
+
+The [=authenticator=] then updates the [=public key credential source/otherUI=]
+of the matching credential.
+
+</div>
+
+<div class="example">
+
+A user has two credentials with [=credential ids=] that [=base64url encode=] to
+`aa` and `bb`. The user deletes the credential `aa` on a [=[WRP]=] provided UI.
+The [=[WRP]=] runs:
+
+```javascript
+PublicKeyCredential.signal({
+    currentCredentials: {
+        rpId: "example.com",
+        userId: "aabbcc",  // user handle, base64url.
+        allAcceptedCredentialIds: [
+            "bb",
+        ]
+    }
+});
+```
+
+If the [=authenticator=] is attached at the time of the report, it deletes or
+hides the [=credential=] corresponding to `aa` from future [=authentication
+ceremonies=].
+
+</div>
+
+Note: [=Authenticators=] may not be attached at the time a report is signalled.
+Therefore, [=[WRPS]=] may choose to signal [=currentCredentials=] periodically,
+e.g. on every sign in.
+
+#### <code>{{PublicKeyCredential/signal()}}</code> algorithm #### {#sctn-signalAlgorithm}
```

Comment:
This seems like it should be more prominent, as it is technically the core of the function specification. I think it should at least come before the other two subsections, but probably even be moved into the parent section? (Just before §5.1.10.1. unknownCredential report)

Reply:
This section is gone now that the PR has been split into three methods.